src/components/LoginForm.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import { Grid, Button, Form, Input } from "semantic-ui-react";
+import { auth } from "../modules/auth";
+import { useHistory } from "react-router-dom";
+import { connect, useDispatch, useSelector } from "react-redux";
```

Comment:
We don't need to connect() if we use redux hooks! The hooks are already connected.